RELEASE_NOTES.md

```diff
@@ -141,29 +144,29 @@ In case of a build failure another link, that opens the `pom.xml` of the failed
 
 #### the m2e-pde editor now supports generation of a feature from a location:
 
-In the wizard it is now possible to request generation of a feature
+In the wizard, it is now possible to request the generation of a feature
 
 ![grafik](https://user-images.githubusercontent.com/1331477/139412713-e0218ff7-642c-4c19-afac-55fca49ef325.png)
 
-If the option is checked the wizard contains a new page to enter basic infos
+If the option is checked the wizard contains a new page to enter basic info
 
 ![grafik](https://user-images.githubusercontent.com/1331477/139412847-5268aaae-de32-472c-b30e-44a8e88728cd.png)
 
 If one likes he can add additional plugins as well that should be mentioned in the feature (but this step is optional)
 
 ![grafik](https://user-images.githubusercontent.com/1331477/139413014-ff0463e2-cd89-41c4-89dd-fe4402be05b2.png)
 
-Afterwards this will end up in the target source and one can add/edit/adjust additional data, effectivly everything the feature.xml supports is allowed here:
+Afterward this will end up in the target source and one can add/edit/adjust additional data, effectively everything the feature.xml supports is allowed here:
 
 ![grafik](https://user-images.githubusercontent.com/1331477/139413236-f04d9b5f-54a5-4240-b83f-86167c7519b2.png)
 
 #### Include and use Maven 3.8.4
 
-Maven 3.8.4 is now used internally in m2e. This allows to benefit from various improvements of recent Maven versions
+Maven 3.8.4 is now used internally in m2e. This allows benefiting from various improvements of recent Maven versions
```

Comment:
The version mentioned here is different from the 3.8.6 mentioned above. I think it should be 3.8.6 here, too